app/controllers/Application.java

```diff
@@ -0,0 +1,15 @@
+package controllers;
+
+import play.*;
```

Comment:
Trebao bi podesiti settings na IntelliJ-u da ne koristi wildcard importe (`*`).

´Editor > Code Style > Java > Imports´ i stavi `Class count to use import with '*'` i `Names count to use static import with '*'` na 99

Reply:
Nakon toga, imaš shortcut da optimizira importe, tj da izbaci ne korištene. Nisam siguran tačno koji je na Windowsu ali to ti je lagano naći

Reply:
Eto ga, mislim da je ovo gotovo, da merge-am na develop?